docs/modules/ROOT/pages/blocking-safe-by-default.adoc

```diff
@@ -57,13 +57,7 @@ code can interact with the event loop on a certain path.
 
 === Execution Strategy
 
-An link:{source-root}/servicetalk-transport-api/src/main/java/io/servicetalk/transport/api/ExecutionStrategy.java[Execution Strategy]
-has two primary purposes:
-
-. Define which interaction paths require offloading.
-. Optionally specify an
-link:{source-root}/servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/Executor.java[Executor]
-to use for offloading. In absence of a specified `Executor`, ServiceTalk will use a default `Executor`.
+The primary purpose of an link:{source-root}/servicetalk-transport-api/src/main/java/io/servicetalk/transport/api/ExecutionStrategy.java[Execution Strategy] is to define which interaction paths require offloading.
```

Comment:
While you are here, please fix a typo in line 55: `deduce` -> `reduce`

Reply:
Not a typo, but I have replaced deduce with determine for clarity.